src/pymodaq/dashboard.py

```diff
@@ -1132,9 +1132,12 @@ def add_move(
         move_forms,
         actuators_modules,
     ) -> DAQ_Move:
-        try:
-            ui_identifier = plug_settings["main_settings", "ui_type"]
-        except KeyError:
+        if plug_settings is not None:
+            try:
+                ui_identifier = plug_settings["main_settings", "ui_type"]
+            except KeyError:
+                ui_identifier = None
```

Comment:
something like would be better for both None lines!

ui_identifier = config('actuator', 'ui')